assets/styles/utilities/_spacing.scss

```diff
@@ -2,21 +2,20 @@
 // Utilities / Spacing
 // ==========================================================================
 
-////
-/// Utility classes to put specific spacing values onto elements. The below loop
-/// will generate us a suite of classes like:
-///
-/// @example
-///   .u-margin-top {}
-///   .u-padding-left-large {}
-///   .u-margin-right-small {}
-///   .u-padding {}
-///   .u-padding-right-none {}
-///   .u-padding-horizontal {}
-///   .u-padding-vertical-small {}
-///
-/// @link https://github.com/inuitcss/inuitcss/blob/512977a/utilities/_utilities.spacing.scss
-////
+//
+// Utility classes to put specific spacing values onto elements. The below loop
+// will generate us a suite of classes like:
+//
+// @example
+//   .u-margin-top {}
+//   .u-padding-left-large {}
+//   .u-margin-right-small {}
+//   .u-padding {}
+//   .u-padding-right-none {}
+//   .u-padding-horizontal {}
+//   .u-padding-vertical-small {}
+//
+// @link https://github.com/inuitcss/inuitcss/blob/512977a/utilities/_utilities.spacing.scss
```

Comment:
If we want to follow [SassDoc](http://sassdoc.com/file-level-annotations/), we should be using `///` instead of `//`.